worker/worker/stream.go

```diff
@@ -53,7 +45,15 @@ func stream(streamCtx *StreamContext) {
 		}
 		// Create a new pgid for the new process, so we don't kill the parent process when ending the stream
 		cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
-		err := cmd.Run()
+
+		err = cmd<.Run()
```

Comment:
This doesn't compile, have you tested the changes locally against a lecture hall stream?